src/IK/src/InverseKinematics.cpp

```diff
@@ -852,6 +794,8 @@ bool HumanIK::initializeJointConstraintsTask(
         {
             lowerLimits[i] = m_kinDyn->model().getJoint(i)->getMinPosLimit(i);
             upperLimits[i] = m_kinDyn->model().getJoint(i)->getMaxPosLimit(i);
+            std::cout << "joint " << m_kinDyn->model().getJointName(i) << " upper lim " << upperLimits[i] << " lower limit "
+                      << lowerLimits[i] << " joint pos: " << m_jointPositions[i] << std::endl;
```

Comment:
I think this can be removed

Reply:
Done with commit https://github.com/ami-iit/biomechanical-analysis-framework/pull/31/commits/2a4606a02a2dcf6ed2fa2564db8d7b83456d2c40